src/QuickMarcEditor/utils.js

```diff
@@ -374,6 +374,17 @@ export const checkControlFieldLength = (formValues) => {
   return undefined;
 };
 
+export const checkDuplicate010Field = (formValues) => {
+  const marcRecords = formValues.records || [];
```

Comment:
Khalilah said that we only need to validate duplicate 010 only in Authority records for now.

Can we call this function in `validateMarcAuthorityRecord` function in utils instead of `onSubmit`?